samples/cpp/benchmark_app/main.cpp

```diff
@@ -200,6 +200,23 @@ void warn_if_no_batch(const benchmark_app::InputsInfo& first_inputs) {
             << slog::endl;
     }
 }
+
+void fuse_mean_scale(ov::preprocess::PrePostProcessor& preproc, const benchmark_app::InputsInfo& app_inputs_info) {
+    for (const std::pair<std::string, benchmark_app::InputInfo>& input_info : app_inputs_info) {
+        if (!input_info.second.mean.empty()) {
+            preproc.input(input_info.first)
+                .preprocess()
+                .convert_element_type(ov::element::f32)
```

Comment:
Do it have a sense to warn user in case when he specify input precision directly using `-ip` and it is differs from `f32` togather with `mean/scale_values`?

Reply:
It still preserves `-ip`. First input is fed with original input precision. Then it is converted to f32 for mean and scale. I think the overhead of type conversion can be treated as the overhead of applying mean/scale. So no warning is needed

Reply:
> It still preserves `-ip`. First input is fed with original input precision. Then it is converted to f32 for mean and scale. I think the overhead of type conversion can be treated as the overhead of applying mean/scale. So no warning is needed

Ok, it has a sense.